test/gtest/uct/test_md.cc

```diff
@@ -376,17 +398,26 @@ UCS_TEST_SKIP_COND_P(test_md, alloc,
     params.mds.count       = 1;
 
     ucs_for_each_bit(mem_type, md_attr().alloc_mem_types) {
-        for (unsigned i = 0; i < 300; ++i) {
-            size = orig_size = ucs::rand() % 65536;
-            if (size == 0) {
+        for (unsigned i = 0; i < iterations; ++i) {
+            size = orig_size = ucs::rand() %
+                               ucs_min(65536, md_attr().max_alloc);
+            aligned_size     = ucs_align_up_pow2(size, sizeof(size_t));
```

Comment:
seems like we don't really need aligned_size. it's used only for max_alloc check